apps/prairielearn/src/lib/code-caller/index.ts

```diff
@@ -53,12 +52,13 @@ export async function init() {
     {
       create: async () => {
         const codeCallerOptions = {
+          dropPrivileges: false,
           questionTimeoutMilliseconds: config.questionTimeoutMilliseconds,
           pingTimeoutMilliseconds: config.workerPingTimeoutMilliseconds,
           errorLogger: logger.error.bind(logger),
         };
 
-        const codeCaller = run(() => {
+        const codeCaller: CodeCallerContainer | CodeCallerNative = run(() => {
```

Comment:
Type inference should make this unnecessary. If you really want to be explicit, you can use the `CodeCaller` type instead.

```suggestion
        const codeCaller = run(() => {
```